pint needs pinning in stable or a fix

The weekly check of our dependency found a problem caused by pint

https://github.com/Open-MSS/MSS/actions/runs/7319432775/job/20050797319#step:9:728


```
self = <matplotlib.axis.YAxis object at 0x7f2c9c54e810>
x = <Quantity([], 'pascal')>

    def convert_units(self, x):
        # If x is natively supported by Matplotlib, doesn't need converting
        if munits._is_natively_supported(x):
            return x
    
        if self.converter is None:
            self.converter = munits.registry.get_converter(x)
    
        if self.converter is None:
            return x
        try:
            ret = self.converter.convert(x, self.units, self)
        except Exception as e:
>           raise munits.ConversionError('Failed to convert value(s) to axis '
                                         f'units: {x!r}') from e
E           matplotlib.units.ConversionError: Failed to convert value(s) to axis units: <Quantity([], 'pascal')>

/opt/conda/envs/mss-stable-env/lib/python3.11/site-packages/matplotlib/axis.py:1772: ConversionError
```

--- a/localbuild/meta.yaml
+++ b/localbuild/meta.yaml
@@ -76,7 +76,7 @@ requirements:
     - PyMySQL >=0.9.3
     - validate_email
     - multidict
-    - pint
+    - pint <=0.22
     - python-socketio >=5
     - python-engineio >=4
     - markdown

